### PR TITLE
Adding My Community

### PR DESCRIPTION
**Pull request by itstalmeez**
I am adding my community to Global Azure

### CODE REVIEW

**Review by noopman — CHANGES_REQUESTED** [review]
This is not right. You need to have your files in a subfolder with your community name. Sorry for the confusion. Please review https://github.com/globalazure/communities/blob/main/README.md

**Comment by itstalmeez** [comment]
Check now

**Comment by itstalmeez** [comment]
@noopman Did you check?

**Comment by noopman** [comment]
Haha, no not yet, I didn't.

File: 2024/Tech Empire/data.json
Comment (noopman) [inline 2024/Tech Empire/data.json:1]: If you have an adress, then maybe you can also add the lon/lat for the location, please? This is how the map finds the location. Compare with https://github.com/globalazure/communities/blob/main/2024/azureskane/data.json

**Review by noopman — CHANGES_REQUESTED** [review]
A few more changes and we're there! Good work!

**Comment by itstalmeez** [comment]
Okay check now

**Comment by itstalmeez** [comment]
@noopman heyyyyyy!!!!!!!! I think so I've done.

**Comment by noopman** [comment]
@itstalmeez your PR contains double the files. BOTH in the /2024 folder AND in your community subfolder. You need to remove the marked files below from this PR, please!

image

**Comment by itstalmeez** [comment]
@noopman Check now

**Review by noopman — APPROVED** [review]
That's it! Perfect, thanks.

**Pull request merged**